examples/plugins/todo-checker/README.md

```
@@ -0,0 +1,207 @@
+
+# Fix issues automatically
+gomdlint fix --plugin-rules document.md
+```
+
+## Development Guide
+
+This plugin demonstrates several patterns for creating gomdlint plugins:
+
+### Plugin Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)